bundler/spec/support/artifice/helpers/compact_index.rb

```diff
@@ -80,7 +80,7 @@ def gems(gem_repo = default_gem_repo)
               CompactIndex::Dependency.new(d.name, reqs)
             end
             checksum = begin
-                         Digest(:SHA256).file("#{gem_repo}/gems/#{spec.original_name}.gem").base64digest
+                         Digest(:SHA256).file("#{gem_repo}/gems/#{spec.original_name}.gem").hexdigest
```

Comment:
why is this changing?

Reply:
Because the actual compact index api uses a hexdigest so we want to use a hexdigest for testing

https://github.com/rubygems/rubygems.org/blob/00ca6264ba74727a24b6021e18937ca7ba5fd91b/app/models/gem_info.rb#L110